### ISSUE TITLE
[FEATURE] Add option to make https enabled by default

### ISSUE THREAD

**Issue by JonathanDoelan** [caprover/caprover#1623]
I have no app that I don’t want to run with `https`. Of course there could be some situations where I don’t want `ssl`.
However it would be nice to have at least an option in the `CapRover > Settings` to automatically enable `https` after a web-app has been created.

**Comment by redwerkz** [comment]
> Also forcing `https` by default would save some time.

enforce security by default

I was wondering too why this needs extra attention by the user. this is a must have. make it opt-out.

**Comment by SpaceIgniter** [comment]
I disagree. I use Caprover Locally and forcing SSL will break my entire environment. It will break any new Local environment as well.

**Comment by JonathanDoelan** [comment]
@SpaceIgniter Then you simply wouldn't turn on this option.

**Comment by guilh22** [comment]
For service like VPN, https is mandatory to correctly set up the service. Having a way to set https by default and to always redirect to https would be amazing (from the captain definition). I also think a place to always set https in the setting could be a great option, but being able to enforce it both ways could be awesome
